src/iuse_actor.cpp

```diff
@@ -4548,24 +4545,41 @@ int sew_advanced_actor::use( player &p, item &it, bool, const tripoint & ) const
     };
 
     uilist tmenu;
-    // TODO: Tell how much thread will we use
-    if( it.charges >= thread_needed ) {
-        tmenu.text = _( "How do you want to modify it?" );
-    } else {
-        tmenu.text = _( "Not enough thread to modify.  Which modification do you want to remove?" );
-    }
+    tmenu.text = _( "How do you want to modify it?" );
 
     int index = 0;
     for( auto cm : clothing_mods ) {
         auto obj = cm.obj();
         item temp_item = modded_copy( mod, obj.flag );
         temp_item.update_clothing_mod_val();
-        bool enab = can_add_mod( obj.flag, obj.item_string );
+
+        bool enab = false;
         std::string prompt;
         if( mod.item_tags.count( obj.flag ) == 0 ) {
-            prompt = string_format( "%s (%d %s)", obj.implement_prompt, items_needed,
-                                    item::nname( obj.item_string, items_needed ) );
+            // Mod not already present, check if modification is possible
+            if( it.charges < thread_needed ) {
+                prompt = string_format( "Can't %s (need %d thread loaded)", obj.implement_prompt, thread_needed );
```

Comment:
```suggestion
                //~ %1$s: modification desc, %2$d: number of thread needed
                prompt = string_format( _( "Can't %1$s (need %2$d thread loaded)" ), obj.implement_prompt, thread_needed );
```

Reply:
Isn't `_()` redundant? It is already called on `prompt` before it is used, on line 4599.

Also, just curious, why is `%1$s` better than `%s`? For code readability/clarity, or some other reason?

Reply:
It allows reordering the strings when translating.

Reply:
> It is already called on prompt before it is used, on line 4599.

`_()` is used to get the translation of a string, and calling it on an already formatted string usually has no effect, so the code on line 4599 is incorrect. I guess it is intended to translate `obj.destroy_prompt` on line 4583, so you may want to fix it by moving the `_()` call on line 4599 to line 4583, around `obj.destroy_prompt`.